routes/authRoutes.js

```diff
@@ -1,6 +1,6 @@
 const express = require("express");
 const router = express.Router();
-const usercontroller = require("../controllers/authController.js");
+const usercontroller = require("../controllers/advertismentController");
```

Comment:
Why is this called `usercontroller` ?

Reply:
Thanks for pointing that out! The require statement for 'authController' was incorrectly named 'advertisementController'. I'll change it to 'authController' to reflect the correct controller.